src/main/php/scriptlet/HttpScriptletRequest.class.php

```diff
@@ -281,6 +281,14 @@ public function getSessionId() {
    * @return string
    */
   protected function importParam($value) {
+
+    // Handle file uploads - these are represented as arrays containg elements for each
+    // file being uploaded
+    // TODO: Do filenames need utf-8 conversion?
```

Comment:
What about the TODO here? The intention of this function is to make sure the incoming data is encoded in internal enconding (utf8 as we're speaking). Do we need to handle filenames as regular strings, or is there never a case a filename would be encoded in such a way (I think it could be, but is it portable?)...